macros/clone_schema.sql

```diff
@@ -0,0 +1,266 @@
+{%- macro clone_schema(schema_one, schema_two, comment_tag='') -%}
+
+    {#/* Copies tables, sequences and views from `schema_one` to `schema_two` if `comment_tag` isn't specified. If `comment_tag` argument is specified, it copies only tables and sequences that have `comment` metadata field equal to the passed value of `comment_tag` argument.
```

Comment:
To be honest description of this macro doesn't include the fact that not passing any value to `comment_tag` arg makes it copy not only tables, sequences and views but functions too. I'll extend it by this details.

Reply:
I haven't covered this `functions` behaviour by tests and consider it as a redundant work for now. We can skip them on Postgres target easily but I still don't check how Snowflake deal with them via CLONE SCHEMA command.